src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.model;
+
+import java.util.Arrays;
+
+public class FruitTransaction {
```

Comment:
The class implementation should not begin with an empty line. Please remove the redundant empty line at the beginning of the class definition.